ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -16,8 +16,34 @@ export XDG_RUNTIME_DIR="${HOME}/run"
 export REGISTRY_AUTH_PREFERENCE=podman # TODO: remove later, used for migrating oc from docker to podman
 mkdir -p "${XDG_RUNTIME_DIR}"
 
-oc registry login
-oc adm release extract --credentials-requests --cloud=aws --to="/tmp/credrequests" "$RELEASE_IMAGE_LATEST"
+echo "RELEASE_IMAGE_LATEST: ${RELEASE_IMAGE_LATEST}"
+
+build_pull_secret="/tmp/build_pull_secret"
+oc registry login --to=${build_pull_secret}
```

Comment:
I am thinking if we can directly merge "${CLUSTER_PROFILE_DIR}/pull-secret" to the default pullsecret generated from `oc registry login`, then make the following oc command directly consume it, but not have to specify the new pullsecret file via a parameter.

Reply:
The benefit of using new `new_pull_secret` is everyone knows this is a combined pull secret, not a default one.